deploy/chart/values.yaml

```diff
@@ -2,7 +2,11 @@ rbacApiVersion: rbac.authorization.k8s.io
 namespace: operator-lifecycle-manager
 catalog_namespace: operator-lifecycle-manager
 operator_namespace: operators
+<<<<<<< HEAD
```

Comment:
git conflict here

Reply:
It gives me great strife to think about how long that was there.